source/features/dim-bots.tsx

```diff
@@ -4,15 +4,24 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-const commitSelector = [
-	/* Commits */
-	'.commit-author[href$="%5Bbot%5D"]',
-	'.commit-author[href$="renovate-bot"]',
-	'.commit-author[href$="scala-steward"]',
-].join(',');
+const commitSelectors = [
+	'actions-user',
+	'bors',
+	'imgbot',
+	'octomerger',
+	'renovate-bot',
+	'rust-highfive',
+	'rust-lang',
+	'scala-steward',
+	'snyk-bot',
+	'web-flow',
+].map(bot => `.commit-author[href$="?author=${bot}"]`);
+
+commitSelectors.push('.commit-author[href$="%5Bbot%5D"]'); // Generic [bot] label
+
+const commitSelector = commitSelectors.join(',');
 
 const prSelector = [
-	/* Issues/PRs */
 	'.opened-by [href*="author%3Aapp%2F"]',
 	'.labels [href$="label%3Abot"]',
 ];
```

Comment:
Should we also expand the list of PR selectors? Or is this impossible due to DOM limitations?

Reply:
Up to @kidonng, who opened the original request

Reply:
I'm not sure what that means? It's looks fine as-is.

Reply:
@cheap-glitch perhaps you meant #5512 😄 sorry I'm dumb